Extension/package.nls.json

```diff
@@ -242,7 +242,8 @@
     "c_cpp.configuration.caseSensitiveFileSupport.markdownDescription": { "message": "If set to `default`, the file system of the workspace is assumed to be case insensitive on Windows and case sensitive on macOS or Linux. If set to `enabled`, the file system of the workspace is assumed to be case sensitive on Windows.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.enhancedColorization.markdownDescription": { "message": "If enabled, code is colorized based on IntelliSense. This setting only applies if `#C_Cpp.intelliSenseEngine#` is set to `default`.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.codeFolding.description": "If enabled, code folding ranges are provided by the language server.",
-    "c_cpp.configuration.hover.description": "If disabled, hover details are no longer provided by the language server.",
+    "c_cpp.configuration.markdownCommentFormatting.description": "If enabled, markdown comment formatting is provided by the language server.",
+    "c_cpp.configuration.hover.description": "Select whether markdown features will be available in comment hover formatting. Default indicates only a subset of markdown features are available.",
```

Comment:
The phrase "comment hover formatting" doesn't really make sense to me, since there's no actual "formatting" being done. It's more like "rendering".